docs/continuous-testing/running-android-unit-tests.md

```diff
@@ -85,11 +84,11 @@ This simple Bash script will trigger your unit test and output the test results
 
 Appcircle has [Test Report Component](https://github.com/appcircleio/appcircle-test-report-component) which can show the result of your tests and code coverage in a beautiful UI.
 
-<NarrowImage src="https://cdn.appcircle.io/docs/assets/test-reports1.png" />
+<Screenshot url="https://cdn.appcircle.io/docs/assets/android-unit-test-report-overview.png" />
 
 You must add this component **after** running your tests so that it can parse test results. Your workflow should look like the below.
 
-<NarrowImage src="https://cdn.appcircle.io/docs/assets/test-reports8.png" />
+<Screenshot url="https://cdn.appcircle.io/docs/assets/android-unit-test-workflow.png" />
```

Comment:
We have missing screenshot here. I will try to fix it... :wrench: 

![Screenshot from 2023-12-01 15-57-39](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/2334a5a1-d437-494d-96c6-c6734ec6f277)